cns/singletenantcontroller/conversion.go

```diff
@@ -59,19 +58,19 @@ func SwiftNodeNetworkConfigListener(cnscli cnsClient) NodeNetworkConfigListenerF
 	}
 }
 
-// CRDStatusToNCRequest translates a crd status to createnetworkcontainer request
-func CRDStatusToNCRequest(status *v1alpha.NodeNetworkConfigStatus) (cns.CreateNetworkContainerRequest, error) {
+// CreateNCRequestFromDynamicNNC translates a crd status to createnetworkcontainer request for dynamic NNC (swift)
+func CreateNCRequestFromDynamicNNC(nnc *v1alpha.NodeNetworkConfig) (cns.CreateNetworkContainerRequest, error) {
```

Comment:
any particular reason we pass a pointer to nnc here? I guess since we don't do a `nil` check then it'll never be `nil`, so we can probably just not use a pointer here

Reply:
cheaper than doing pbv, but nil checks are a good idea

Reply:
actually, I want this communication to eventually become channels, so changing to pbv